Add note regarding presence reset action (#279)
2600hz/kazoo-crossbar: doc/presence.md

    'http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/presence/{EXTENSION}'
```

**Note about *reset* request**: actually, *reset* is not what it seems to represent. *reset* action

> request subscribers to replace any state they have for the given presence\_id ({EXTENSION} in this case)

Reviewer: ```suggestion
request *presence* listeners such as *kamailio* to reset any state they have for the given presence\_id ({EXTENSION} in this case)
```

with the presence state for the given presence_id within `{ACCOUNT_DB}/manual_presence` document, if any.

> 

Reviewer: we're just saying, `hey! reset this`. remove this

In short, *reset* acts like a *flush* command with `{ACCOUNT_DB}/manual_presence` document as the data source.

> 

Reviewer: ```suggestion
In short, *reset* acts like a *flush* command

```
reset works for all/any presence id, not necessarily on `manual_presence`
